### PR TITLE
Increase horizontal page padding

### PR DESCRIPTION
**Pull request by silverwind**
Add a bit more empty space on left and right side of page content for a more pleasant viewing experience. Also tweaked the mobile navbar to match.

Before:
Screenshot 2023-03-16 at 00 58 23

After:
Screenshot 2023-03-16 at 00 58 37

Mobile Navbar:
Screenshot 2023-03-16 at 01 05 12

### CODE REVIEW

**Comment by silverwind** [comment]
This will require either the CSS backport or a manual backport.

File: web_src/css/base.css [hunk 1]
```diff
@@ -1285,7 +1285,14 @@ a.ui.card:hover,
 .following.bar #navbar {
   width: 100vw;
   min-height: 52px;
-  padding: 0 0.5rem;
+  padding: 0 16px;
+}
+
+@media (max-width: 767px) {
+  .following.bar #navbar {
+    padding-left: 4px;
```
Comment (HesterG) [inline web_src/css/base.css:1293]: just curious why 4px is added to the padding-left, the visual effect looks good though.

Comment (silverwind) [reply web_src/css/base.css:1293]: It's just a pixel hack on top of fomantic's silly `.92857143em` padding inside the navbar box 😉.

Comment (HesterG) [reply web_src/css/base.css:1293]: lol I see.

**Review by HesterG — APPROVED** [review]

**Review by lunny — APPROVED** [review]

**Comment by techknowlogick** [comment]
ping lg-tm

**Pull request merged**